p2p/relay.go

```diff
@@ -69,7 +69,7 @@ func NewRelayReserver(tcpNode host.Host, relay *MutablePeer) lifecycle.HookFuncC
 
 			refresh := time.After(refreshDelay)
 
-			timer := time.NewTimer(time.Millisecond * 100)
+			timer := time.NewTicker(time.Millisecond * 500)
```

Comment:
```suggestion
			ticker := time.NewTicker(time.Millisecond * 500)
```

Reply:
why updated to 500ms?

Reply:
100ms is very fast, 500ms should be fine. It was 1s originally. We decreased to 100ms to debug.